tdconfig.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
This is a minimal tsconfig.json for typescript that can be used to generate the docs